demo/slimfacenet/models/calc_flops.py

```diff
@@ -0,0 +1,214 @@
+from collections import OrderedDict
```

Comment:
Why not use `slim.analysis.flops` to calc FLOPs?

Reply:
calc_flops can be merged later